src/main/asciidoc/_chapters/architecture.adoc

```diff
@@ -3011,20 +3011,6 @@ Instead you can change the number of region replicas per table to increase or de
       Whether asynchronous WAL replication to the secondary region replicas is enabled or not. If this is enabled, a replication peer named "region_replica_replication" will be created which will tail the logs and replicate the mutations to region replicas for tables that have region replication > 1. If this is enabled once, disabling this replication also requires disabling the replication peer using shell or Admin java class. Replication to secondary region replicas works over standard inter-cluster replication.
     </description>
 </property>
-<property>
-  <name>hbase.region.replica.replication.memstore.enabled</name>
```

Comment:
This config is removed?

Reply:
Yes, this config is added by HBASE-13184, which is a subtask of HBASE-13063, but the HBASE-13063 seems does not used this config, and I could not search any code use this config, instead per-table property `REGION_MEMSTORE_REPLICATION` is used.